src/main/java/dan200/computercraft/core/logger/Logger.java

```diff
@@ -0,0 +1,65 @@
+package dan200.computercraft.core.logger;
+
+import javax.annotation.Nonnull;
+import java.util.logging.Level;
+
+public class Logger
+{
+    private static ILogger instance;
+
+    @Nonnull
+    public static ILogger getInstance()
```

Comment:
Not really. It could happen, but I'm not especially worried if there are two `StdoutLogger`s created - it isn't required to be a singleton and their constructor hasn't any side effects.